overlord/overlord_test.go

```diff
@@ -704,35 +737,33 @@ func (ovs *overlordSuite) TestEnsureLoopPruneAbortsOld(c *C) {
 		return &state.Retry{}
 	}, nil)
 
-	markSeeded(o)
-
 	st := o.State()
 	st.Lock()
 
 	// start of operation time is a year ago
 	opTime := time.Now().AddDate(-1, 0, 0)
 	st.Set("start-of-operation-time", opTime)
 
-	st.Unlock()
-	c.Assert(o.StartUp(), IsNil)
-	st.Lock()
```

Comment:
what's the reason for moving this and markSeeded, I'm missing something. it looks a bit odd

Reply:
Sorry about that, no good reason, as I was iterating over this test to fix the flakiness I did some modifications that were not really relevant. Reverted.